### ISSUE TITLE
Add configuration option to enable/disable suspect_records log

### ISSUE THREAD

**Issue by orangejulius** [pelias/wof-admin-lookup#22]
This file can grow quite large (5GB+ on full imports), and is probably not useful to run for every build.

Update: in line with https://github.com/pelias/dbclient/pull/22, a better change feels like logging to stdout, but at the debug level, which we would not by default enable all the time.

**Comment by avulfson17** [comment]
Lets grab the latest version of this file from prod_build before this ticket is addressed

**Comment by orangejulius** [comment]
Here are the latest logs from prod_build:


[suspect_wof_records.openstreetmap.txt](https://github.com/pelias/wof-admin-lookup/files/399799/suspect_wof_records.openstreetmap.txt)
[suspect_wof_records.openaddresses.txt](https://github.com/pelias/wof-admin-lookup/files/399797/suspect_wof_records.openaddresses.txt)
[suspect_wof_records.geonames.txt](https://github.com/pelias/wof-admin-lookup/files/399798/suspect_wof_records.geonames.txt)

**Comment by dianashk** [comment]
cc @stepps00 @thisisaaronland @nvkelso
The files in the previous comment were generated during our latest build. We will turn off their generation for a bit, but if you'd like to rerun this again in the future let us know.

The files contain all the points we've come across that didn't come back with a country during point in polygon lookup. The format is a bunch of lines, each one being well formed json. The json object on each line is as follows. In some cases we are unable to find any admin areas for a given point, in which case you won't see all those results, but instead an empty array.

```
{
  "centroid": {
    "lon": 151.29787,
    "lat": -33.715306
  },
  "result": {
    "locality": [
      {
        "id": 101931371,
        "name": "Narrabeen"
      }
    ],
    "neighbourhood": [
      {
        "id": 85769849,
        "name": "Narrabeen"
      }
    ],
    "localadmin": [
      {
        "id": 404225219,
        "name": "Narrabeen"
      }
    ],
    "county": [
      {
        "id": 102049493,
        "name": "Warringah"
      }
    ],
    "region": [
      {
        "id": 85681545,
        "name": "New South Wales"
      }
    ]
  },
  "level": "info",
  "message": "no country"
}
```

**Comment by thisisaaronland** [comment]
Interesting. We can probably backfill a number of those by testing `iso:country` or `wof:country`

```
python ./wof-pip-assign-hierarchy -R /usr/local/data/whosonfirst-data -A /usr/local/data/whosonfirst-data -i 101931371
# hierarchy (PIP stuff)
[{u'continent_id': 102191583,
  u'country_id': -1,
  u'localadmin_id': 404225219,
  u'locality_id': 101931371,
  u'region_id': -1}]
# iso:country
AU
```

**Comment by thisisaaronland** [comment]
A dry-run, checking the `wof-data` repo for things missing a country ID or a country ID of `-1` yields:

```
update from PIP 521 from country 13072 unable to update 1555
```

This does _not_ account for things with multiple hierarchies. I will run the backfill for real and commit the changes, shortly.

**Comment by dianashk** [comment]
Thanks @thisisaaronland for digging into this.

**Comment by orangejulius** [comment]
Fixed in #63!
